inference-engine/src/mkldnn_plugin/config.cpp

```diff
@@ -33,6 +33,9 @@ Config::Config() {
     streamExecutorConfig._threadBindingType = InferenceEngine::IStreamsExecutor::CORES;
 #endif
 
+    if (!with_cpu_x86_bfloat16())
```

Comment:
Why you moved this condition from updateProperties()?

Reply:
updateProperties() just reflects of internal state of CPU plugin and shouldn't change settings. A proper place is in ctor, which sets initial behaviour on different platforms.